### DISCUSSION TITLE
Select 值自动切换问题

### DISCUSSION THREAD

**Discussion by zhangtuoyu** [dotnetcore/BootstrapBlazor#4872] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Select控件放到MainLayout中,每次切换页面都会自动选择到第一个选项
但SelectGeneric 不会有这样的问题

### Expected Behavior

我希望切换页面,不会导致 Select 触发选择事件来重置为第一个选项值.

### Interactive render mode

Interactive Server (Interactive server-side rendering (interactive SSR) using Blazor Server)

### Steps To Reproduce

[BootstrapBlazorApp1.zip](https://github.com/user-attachments/files/18159589/BootstrapBlazorApp1.zip)

### .NET Version

NET9.0

**Comment by bb-auto** [comment]
@zhangtuoyu Thank you for reporting. We will give triage later.
